### ISSUE TITLE
Help file incorrectly installed by the Debian package

### ISSUE THREAD

**Issue by mgedmin** [gtimelog/gtimelog#111]
When I press F1, I get yelp complaining that `help:gtimelog/index` does not exist.

**Comment by mgedmin** [comment]
`dpkg -L gtimelog` shows

```
/usr/share/help
/usr/share/help/C
/usr/share/help/C/shortcuts.page
/usr/share/help/C/figures
/usr/share/help/C/figures/gtimelog.png
/usr/share/help/C/legal.xml
/usr/share/help/C/index.page
```

These files should show up in `/usr/share/help/C/gtimelog/*.page`!

**Comment by mgedmin** [comment]
Should be fixed in 0.11-4mg2 that I've just uploaded to the PPA.
